### PR TITLE
merge dev into main branch.

### PR DESCRIPTION
**Pull request by GuilhermeFelipeCoelho**
remade html -> angular
refactor the folder to the project root

### CODE REVIEW

File: src-tauri/tauri.conf.json [hunk 1]
```diff
@@ -30,7 +34,7 @@
     "bundle": {
       "active": true,
       "targets": "all",
-      "identifier": "com.tauri.dev",
+      "identifier": "com.backup.save",
```
Comment (pedro-ryan) [inline src-tauri/tauri.conf.json:37]: "com.backup.save" is the name of the project ?

Comment (Guipeee) [reply src-tauri/tauri.conf.json:37]: oh yes thanks no thanks arigato yes, no. @pedro-ryan @PotenzaFelip

Comment (GuilhermeFelipeCoelho) [reply src-tauri/tauri.conf.json:37]: This is a temp name for the project.

**Review by pedro-ryan — APPROVED** [review]
ok, I think everything is ok

**Pull request merged**